### ISSUE TITLE
Fracture in parallel

### ISSUE THREAD

**Issue by rimoli** [gahansen/Albany#90]
It would be great to have adaptive insertion of cohesive elements for fracture in parallel. More than an issue this is a feature request. I could transfer all my research activities to LCM if this feature was available.

**Comment by lxmota** [comment]
@rimoli @jtostie @jwfoulk Yes, this is a long-standing issue that is in my personal wall of shame. I will find some time to check the status of the implementation and report back. Thanks for opening this issue.

**Comment by rimoli** [comment]
@lxmota @jtostie @jwfoulk I am open to help, at least if you guys want to brainstorm solutions.

**Comment by lxmota** [comment]
@rimoli @jtostie @jwfoulk My recollection is that there was a conflict between STK and the way the worksets were managed when making topology changes to the mesh. It has been so long that I'm sure things have changed. Let me make an assessment of the status of the implementation to determine what path to follow.
